### DISCUSSION TITLE
Use with Macports install of Borg?

### DISCUSSION THREAD

**Discussion by bunk3m** [borgbase/vorta#2175] [FAQ] [answered]
I was about to install 0.10.2 when I saw the 0.10.3 announcement.  I have installed Borg using Macports.

I see that for version 0.10.3 that the Homebrew installed Borg is now hard coded?  Will that cause a problem for users who have installed Borg using Macports?  The Macports install location for Borg is included in the $PATH command before this hard coded change.  Will 0.10.3 change this so the Macports installed version won't be found?

**Comment by m3nu** [comment, marked as answer]
It shouldn't be hard-coded and the Homebrew path is simply appended to the existing PATH, if it exists. If it doesn't, a minimal default is used.

Best if you check if there is really an issue and let us know the location of your `borg` binary. Then we can potentially make a change to add it too, if it's not a standard PATH.

**Reply by bunk3m** [reply]
Thanks @m3nu .

I am running v0.10.3 and it appears to be working fine.  It is using the version in `/Applications/Vorta.app/Contents/Resources/borg-dir/borg.exe
`
My installed version of borg from Macports is at `/opt/local/bin/borg`

As it is working, I don't really care if it uses the Vorta installed or Macports.

Thanks for your help and suggestion.
